### DISCUSSION TITLE
Add permissions for comment / explain

### DISCUSSION THREAD

**Discussion by rijkvanzanten** [directus/directus#3118] [Feature Requests]
### Comment:

- Allowed (all)
- Not allowed (none)
- ??? based on permissions condition?

### Explain: 

- Allowed (all)
- Not allowed (none)
- ??? Forced – always has to be provided
- ??? based on condition?

@benhaynes

**Comment by benhaynes** [comment]
Yes, this is exactly right. Would we include this under normal permissions? 

If we go that route, it would be good to have a global fallback per role.

If we don't, then I feel like it's easier to set up... but the comment/explained permissions will end up just being set per collection anyway.

Basically it's just two different ways of doing the same thing.

Btw, "permissions" and "force" would be the same thing for _explain_. You're basically choosing when someone _needs_ to explain, otherwise it's basically just a comment, and we can use those permissions.

**Reply by nechdie** [reply]
Does this feature (force explain) still exist in Directus 9? I see it referenced in the V8 docs, but not in V9. If not, is there a similar in-built functionality or would I need to write one by creating a hook?

**Reply by benhaynes** [reply]
No, it has never existed in any version, it was a concept that hasn't yet made it into production.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
